portf.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <title>GintsMisins Portfolio</title>
+  <link rel="stylesheet" href="css/webstyle.css">
+  <link rel="stylesheet" media="screen and (max-width: 500px)" href="css/mmobile.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <h3>GintsMisins</h3>
+    <nav>
+      <ul class="hide">
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+      </ul>
```

Comment:
- [ ] In desktop mode we should see the links for the **contact & about & portfolio**, but there is just the **hamburger menu**.

- **Your Navigation**

![your_logo_header](https://user-images.githubusercontent.com/61562626/134771911-2c7c24aa-1a33-49cd-9162-909af70afb13.jpeg)


- **Figma nav-bar desktop mode**

![logo_header](https://user-images.githubusercontent.com/61562626/134771947-4cd48147-6fa1-4979-9e4e-80c582ae56f8.jpeg)

Reply:
you make a comments in portf.html but screen from index.html